runtime/src/main/java/io/quarkiverse/resteasy/problem/jackson/JacksonProblemModuleRegistrar.java

```diff
@@ -13,9 +13,15 @@ public final class JacksonProblemModuleRegistrar implements ObjectMapperCustomiz
 
     @Override
     public void customize(ObjectMapper mapper) {
-        SimpleModule module = new SimpleModule("RFC7807 problem");
-        module.addSerializer(HttpProblem.class, new JacksonProblemSerializer());
-        mapper.registerModule(module);
+        registerProblemModule(mapper);
+    }
+
+    public static ObjectMapper registerProblemModule(ObjectMapper mapper) {
```

Comment:
`public` for tests only 😜 

You made duplication on `uriOrThrow`/`intOrThrow` yet only for sake of single use in test you "expose API 😉 " in Quarkus Bean.

My preference is to duplicate it in test.

But since this PullReq is "so good" this is the only thing I can catch really 😄 

Good job❕

Reply:
Agree, I took it a bit too far. Reverted this.